docs/workshop.md

```diff
@@ -761,19 +761,23 @@ You have previously created an Azure Function which reacts to cache expiry by li
 
 This new Azure Function will power an API for retrieving a user's browsing history, e.g. which products the users have viewed recently.
 
-But before delving into the Azure Function's code, let's first take a look at the Streams you currently have at disposal.
+![Example of sequence diagram to introduce history-func](./assets/history-func-usage-sequence-diagram.png)
 
 ### Inspecting product views' stream
 
-First thing first, let's take a look at the data currently available on your Azure Cache for Redis instance.
+First thing first, let's take a look at the Streams currently available on your Azure Cache for Redis instance.
 
-There is a variety of tools which allows you to inspect Redis like the integrated [Redis Console][redis-console] and also the fully-featured GUI [RedisInsight][redis-insight].
+The goal is to locate the [Redis Stream][redis-streams] in which the `catalog-api` is adding new items whenever a user views a product. Afterwards we need to inspect that stream and take a look at the events/items added to it.
+
+To do this, there is a variety of tools that you can use to inspect Redis data like the integrated [Redis Console][redis-console] and also the fully-featured GUI [RedisInsight][redis-insight].
 
 
 <div class="task" data-title="Task">
 
->  - Locate the stream where `catalog-api` publishes product viewing events and inspect its contents using the `Redis Console` from the Azure portal.
->  - View some products in the Web App and make sure that you see new items in the stream
+>  - View some products in the Web App (or by directly calling the `/products/:id` endpoint from the `catalog-api`) to generate items in the stream.

```

Comment:
tu peux mettre un exemple de curl pour montrer le call /products/:id par exemple

Reply:
I added a reference to Lab1 where we interact with the Catalog API. With the deployment of the Web App this step of the task should be greatly simplified.